ageCalculator.js

```diff
@@ -0,0 +1,4 @@
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture + ".");
```

Comment:
You've done this exercise excellently. Just one way you can improve on is to learn about [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). It's a much cleaner way to write console.logs Eg.
```suggestion
console.log(`You will be ${age} years old in ${yearFuture}.`);
```